types/svg-parser/index.d.ts

```diff
@@ -1,14 +1,24 @@
-// Type definitions for svg-parser 1.0
+// Type definitions for svg-parser 2.0
 // Project: https://gitlab.com/Rich-Harris/svg-parser#README
 // Definitions by: mrmlnc <https://github.com/mrmlnc>
+//                 Joel Shinness <https://github.com/JoelCodes>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.1
 
-export interface Node {
-    name: string;
-    attributes: Record<string, string | number>;
-    children: Node[];
+export interface TextNode {
+    type: 'text';
+    value?: string | boolean | number;
+}
+
+export interface ElementNode {
+    type: string;
+    tagName?: string;
+    properties?: {};
```

Comment:
We can restrict the types in this location. The `properties` can be `Record<string, string | number>`.

Reply:
That follows.  I've updated this to become more specific.